EventListener/VersionListener.php

```diff
@@ -53,16 +53,18 @@ public function onKernelRequest(GetResponseEvent $event)
         }
 
         $this->version = $this->versionResolver->resolve($request);
-        if (false === $this->version && null !== $this->defaultVersion) {
+        if (false === $this->version && null !== $this->defaultVersion && !$request->attributes->has('version')) {
             $this->version = $this->defaultVersion;
         }
```

Comment:
That's alright, it's not much to do and I won't forget it if I do it now :stuck_out_tongue: